### PR TITLE
Row entry box

### PR DESCRIPTION
**Pull request by fotakis08**

### CODE REVIEW

File: data-table/table.js [hunk 1]
```diff
@@ -310,6 +310,33 @@ function addSingleRow(config, rowIndex, content) {
     }
 }
 
+// eslint-disable-next-line valid-jsdoc
```
Comment (artoonie) [inline data-table/table.js:313]: Curious why the jsdoc isn't valid / can it be made valid?

File: data-table/table.js [hunk 2]
```diff
@@ -310,6 +310,33 @@ function addSingleRow(config, rowIndex, content) {
...
+ *  If the box value requires less number of rows than already exists,
+ *  remove the excess rows from the table starting from the last row
+ *  If additional rows are needed, create the additional rows starting from the last row's index
+ *
+ *  @param {object} config  - table configuration object
+ *  @param {number} boxValue - the box entry value
+ *  @returns {undefined}     - Doesn't return anything
+ */
+
+// eslint-disable-next-line no-unused-vars
```
Comment (artoonie) [inline data-table/table.js:327]: what is unused?

File: data-table/table.js [hunk 3]
```diff
@@ -310,6 +310,33 @@ function addSingleRow(config, rowIndex, content) {
...
+function onRowEntryBoxEnter(config, boxValue) {
+    let table = document.getElementById(config.tableIds.tableElementId);
+    let numRows = table.rows.length; // Get length row right now.
+    // Delete rows that are excess of the box value from the bottom of the table.
+    if(numRows > boxValue) {
+        deleteRows(config, numRows - boxValue, numRows);
+    }else {
```
Comment (artoonie) [inline data-table/table.js:334]: [nit] add a space before `else`

**Review by artoonie — APPROVED** [review]
Tests would be nice too :)

**Review by brmighell — CHANGES_REQUESTED** [review]
Doesn't work yet - will re-review after pair programming on Nov. 1st

**Pull request closed without merging**